navigation.yaml

```diff
@@ -1616,10 +1616,12 @@ ubuntu1604:
 certification:
   title: Hardware
   path: /certified
+  persist: True
+  match_query: True
 
   children:
     - title: Overview
-      path: /certified
+      path: /certified?
```

Comment:
Why does this have a question mark on the end?

Reply:
`request.full_path` doesn't match without the question mark (query string).